.ci/setup_env_centos.sh

```diff
@@ -8,11 +8,15 @@
 set -e
 
 cidir=$(dirname "$0")
-source "/etc/os-release"
+source "/etc/os-release" || "source /usr/lib/os-release"
 source "${cidir}/lib.sh"
 
 # Obtain CentOS version
-centos_version=$(grep VERSION_ID /etc/os-release | cut -d '"' -f2)
+if [ -f /etc/osrelease ]; then
```

Comment:
typo `/etc/os-release` preventing have a value on `$centos_version`

Reply:
Pushed the fix @chavafg PTAL Thx.

Reply:
Good catch @chavafg! :smile: